src/character_controller.h

```diff
@@ -237,15 +237,18 @@ typedef struct character_command_s
     uint16_t    sprint : 1;
 }character_command_t, *character_command_p;
 
-typedef struct character_response_s
+typedef struct character_state_s
 {
     int8_t      vertical_collide;
     int8_t      horizontal_collide;
-    uint16_t    kill : 1;
-    uint16_t    burn : 1;
     uint16_t    slide : 2;
     uint16_t    step_z : 2;
-}character_response_t, *character_response_p;
+    uint16_t    kill : 1;
```

Comment:
yep, first comments, maybe later it will be more clear what to do with architecture